src/app/signin/form/index.tsx

```diff
@@ -33,25 +32,21 @@ const SignInForm = () => {
     setIsOpen(false);
   };
 
-  const handleForgotPassword = () => {};
+  const handleForgetPassword = () => {};
 
-  const { onSubmit, response } = SignInHandler(setIsOpen, isRememberChecked);
+  const { onSubmit, response } = SubmitHandler(setIsOpen, isRememberChecked);
```

Comment:
first letter should not be capitalized (file and function)

Reply:
the import and function call can change, but the component itself can't since they are using react hooks inside them.